pyproject.toml

```
@@ -287,7 +296,7 @@ filterwarnings = [
 # files or test ids are given in the command line when executing pytest from the rootdir
 # directory. File system paths may use shell-style wildcards, including the recursive **
 # pattern.
-testpaths = ["src", "tests"]
+testpaths = ["packages/lmi/tests", "src", "tests"]
```

```suggestion
testpaths = ["packages/lmi", "src", "tests"]
```

We want to do doctests in `lmi` too